diff --git a/config.def.h b/config.def.h
index 9efa774..bbc576f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,34 +1,34 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance */
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const unsigned int borderpx  = 2;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
-static const int showbar            = 1;        /* 0 means no bar */
-static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const int showbar            = 0;        /* 0 means no bar */
+static const int topbar             = 0;        /* 0 means bottom bar */
+static const char *fonts[]          = { "Cascadia Mono:style=Bold:size=17" };
+static const char dmenufont[]       =   "Cascadia Mono:style=Bold:size=17";
+static const char black[]           = "#111111";
+static const char white[]           = "#eeeeee";
+static const char gray[]            = "#333333";
+static const char blue[]            = "#77AAAA";
 static const char *colors[][3]      = {
-	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	/*               fg      bg     border   */
+	[SchemeNorm] = { white,  black, gray },
+	[SchemeSel]  = { blue,   black, blue },
 };
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "1", "2", "3", "4" };
 
 static const Rule rules[] = {
 	/* xprop(1):
 	 *	WM_CLASS(STRING) = instance, class
 	 *	WM_NAME(STRING) = title
 	 */
-	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+	/* class       instance           title      tags mask     isfloating   monitor */
+	{  NULL,      "chromium",         NULL,      1 << 1,       0,           -1 },
+	{  NULL,      "libreoffice",      NULL,      1 << 2,       0,           -1 },
+	{  NULL,      "gimp",             NULL,      1 << 3,       0,           -1 },
 };
 
 /* layout(s) */
@@ -45,7 +45,7 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -57,17 +57,30 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", black, "-nf", white, "-sb", black, "-sf", blue, NULL };
 static const char *termcmd[]  = { "st", NULL };
 
+static const char *brightness_dec[] = { "brightnessctl", "-q", "set", "10%-", NULL };
+static const char *brightness_inc[] = { "brightnessctl", "-q", "set", "10%+", NULL };
+static const char *volume_dec[] = { "wpctl", "set-volume", "@DEFAULT_SINK@", "10%-", NULL };
+static const char *volume_inc[] = { "wpctl", "set-volume", "@DEFAULT_SINK@", "10%+", NULL };
+static const char *screenshot_active[] = { "sh", "-c", "maim -u ~/Downloads/$(date +%s).png", NULL };
+static const char *screenshot_region[] = { "sh", "-c", "maim -u -s ~/Downloads/$(date +%s).png", NULL };
+
 static const Key keys[] = {
 	/* modifier                     key        function        argument */
-	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_a,      spawn,          {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY,                       XK_Left,   spawn,          {.v = brightness_dec} },
+	{ MODKEY,                       XK_Right,  spawn,          {.v = brightness_inc} },
+	{ MODKEY,                       XK_Down,   spawn,          {.v = volume_dec} },
+	{ MODKEY,                       XK_Up,     spawn,          {.v = volume_inc} },
+	{ MODKEY,                       XK_s,      spawn,          {.v = screenshot_active} },
+	{ MODKEY|ShiftMask,             XK_s,      spawn,          {.v = screenshot_region} },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
-	{ MODKEY,                       XK_i,      incnmaster,     {.i = +1 } },
+	{ MODKEY,                       XK_e,      incnmaster,     {.i = +1 } },
 	{ MODKEY,                       XK_d,      incnmaster,     {.i = -1 } },
 	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} },
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
@@ -85,15 +98,10 @@ static const Key keys[] = {
 	{ MODKEY,                       XK_period, focusmon,       {.i = +1 } },
 	{ MODKEY|ShiftMask,             XK_comma,  tagmon,         {.i = -1 } },
 	{ MODKEY|ShiftMask,             XK_period, tagmon,         {.i = +1 } },
-	TAGKEYS(                        XK_1,                      0)
-	TAGKEYS(                        XK_2,                      1)
-	TAGKEYS(                        XK_3,                      2)
-	TAGKEYS(                        XK_4,                      3)
-	TAGKEYS(                        XK_5,                      4)
-	TAGKEYS(                        XK_6,                      5)
-	TAGKEYS(                        XK_7,                      6)
-	TAGKEYS(                        XK_8,                      7)
-	TAGKEYS(                        XK_9,                      8)
+	TAGKEYS(                        XK_u,                      0)
+	TAGKEYS(                        XK_i,                      1)
+	TAGKEYS(                        XK_o,                      2)
+	TAGKEYS(                        XK_p,                      3)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
 };
 
